Automattic/jetpack
# AI Excerpt: introduce AI setting toggle to panel

~Needs https://github.com/Automattic/jetpack/pull/33083 and D121260-code to be able to set the `model`~

Fixes https://github.com/Automattic/jetpack/issues/32878

This PR adds an AI Setting toggle panel to the AI Excerpt (beta) feature. This panel allows set addition settings when the user requests the excerpt suggestion.

Panel initial view | AI settings on
---------|----------
<img width="289" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/90a2ed23-07cf-470b-bfd7-b5c42b54dd06"> |  <img width="315" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/62f4207e-645c-467d-a3fa-fba0272273a6">

## Proposed changes:

* AI Excerpt: introduce AI setting toggle panel

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Open the post sidebar
* Look at the AI Excerpt panel
* Play with the Settings toggle, as the video below shows 

https://github.com/Automattic/jetpack/assets/77539/c9fa9641-caba-444d-a768-e51c957336c5

* Confirm that the generated code follows the settings defined by the user
* You can confirm how the settings are passed to it by examining the request payload, by using the network tab of your dev browser. Chrome:
  * Open the dev tools
  * Select the Network tab
  * Filter the request by `ai-query`
  * Take a look at the `payload`:

<img width="831" alt="Screenshot 2023-09-15 at 11 48 02" src="https://github.com/Automattic/jetpack/assets/77539/041ff091-cfef-4a2e-85d3-7d2915e05d95">
